dataprep/data_connector/implicit_database.py

```diff
@@ -230,10 +232,9 @@ def __init__(self, config_path: Union[str, Path]) -> None:
             if table_config_path.suffix != ".json":
                 # ifnote non json file
                 continue
-
             with open(table_config_path) as f:
                 table_config = jload(f)
-
+                self.config = table_config
```

Comment:
What's this for? If you need to create a new instance variable, please also define it in the class header for easy reading.